apps/console/src/features/core/hooks/use-routes.tsx

```diff
@@ -162,7 +162,7 @@ const useRoutes = (): useRoutesInterface => {
             getAppViewRoutes(commonConfig.useExtendedRoutes),
             featureConfig,
             allowedScopes,
-            true,
+            !legacyAuthzRuntime,
```

Comment:
shall we use a new config under legacy_mode

Reply:
Added